server/index.js

```diff
@@ -78,11 +78,13 @@ app.post('/create', function (req, res) {
       log('Response received from pillar:')
       log(response)
 
-      builder.buildWidget(req.body, false).then(function (code) {
-        return upload(response.data.id, code)
+      builder.buildWidget(req.body, false).then(code => {
```

Comment:
it's just one statement so it can be replaced with

`code => Promise.all([upload(response.data.id, code, './templates/iframe-form.pug'), code]`